pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-signald/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchgit, fetchFromGitHub, pidgin, glib, json-glib, signald }:
+
+stdenv.mkDerivation rec {
+  pname = "purple-signald";
+  version = "0.11.0";
+
+  src = fetchgit {
+    url = "https://github.com/hoehermann/libpurple-signald.git";
+    rev = "ebd7abf91135d2b8f77752ddaffdd989982e55ec";
+    sha256 = "sha256-2LiHjVRBwdPbfravIVM+gvsh3Gq4bhjtRD6eWAbkWmc=";
+  };
+
+
+  buildInputs = [
+    pidgin
+    json-glib
+    signald
+  ];
+
+  PKG_CONFIG_PURPLE_PLUGINDIR = "${placeholder "out"}/lib/purple-2";
+  PKG_CONFIG_PURPLE_DATADIR = "${placeholder "out"}/share";
+
+  installFlags = [ "DESTDIR=$(out)" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/hoehermann/libpurple-signald";
+    description = "Signald plugin for Pidgin";
+    license = licenses.gpl3;
```

Comment:
This needs to be changed according to [this](https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206).